one_fm/templates/pages/applicant_docs.py

```diff
@@ -387,17 +387,35 @@ def send_applicant_doc_magic_link(job_applicant, applicant_name, designation):
 
 @frappe.whitelist(allow_guest=True)
 def update_job_applicant(job_applicant, data):
-    doc = frappe.get_doc('Job Applicant', job_applicant)
+    doc = update_application_function(job_applicant, data)
+    doc.docstatus = 1
```

Comment:
Job Applicant is not submittable doctype, If we set docstatus = 1, then user can t edit the job applicant record.

Reply:
Changes has been made to this effect